ruby/exercises/contador_de_vocales.rb

```diff
@@ -2,4 +2,19 @@
 # Ejemplo:
   # 'Ruby On Rails Course' => { a: 1, e: 1, i: 1, o: 2, u: 2 }
   # 'Gerardo' => { a: 1, e: 1, i: 0, o: 1, u: 0 }
-  # 'BCDF' => { a: 0, e: 0, i: 0, o: 0, u: 0 }
\ No newline at end of file
+  # 'BCDF' => { a: 0, e: 0, i: 0, o: 0, u: 0 }
+  def cuentavocales!(stg)
+    vocales = {a: 0, e: 0, i: 0, o: 0, u: 0}
+    stg.chars.each do |chr|   
+      chr = chr.downcase
```

Comment:
adicional a lo comentado creoq ue estaria bien aplicar adicionalmente el metodo `to_sym` para evitar realizar las conversiones en la linea 10
```suggestion
      aux_chr = chr.downcase.to_sym
```